frontend/src/app/components/recipe-list/recipe-list.component.scss

```diff
@@ -12,16 +13,34 @@
         border-bottom: 1px solid $light-border-subtle;
         transition: background-color 0.3s ease;
 
+        .mat-mdc-icon-button {
+            visibility: visible;
```

Comment:
You are right, the hover effect isn't working with bootstrap classes.
But you could put the &:hover {...} inside the media query because it only applies to md and up.